src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-left: 4px;
+  }
+
+  &__star:nth-child(-n + 4) {
```

Comment:
The logic for setting the active stars is hard-coded. It will always set the first 4 stars as active. If you want to make your stars component reusable, you should consider a different approach. For example, you could use a modifier class like `.stars--4` on the `.stars` element to set 4 stars as active. Then in your CSS, you could use the `:nth-child` selector in combination with the `.stars--4` class to set the correct stars as active.